spec/checkIssueLabelsSpec.js

```diff
@@ -0,0 +1,207 @@
+      const body = (
+        'Hi @' + user + ', thanks for proposing this as a good first issue. ' +
+        'Looping in @' + whitelist.teamLeads.onboardingTeam +
+        ' to confirm, removing the label until he does so.');
+
+      expect(octokit.issues.createComment).toHaveBeenCalledWith({
+        issue_number: payload.issue.number,
+        body: body,
+        owner: payload.repository.owner.login,
+        repo: payload.repository.name,
+      });
+    });
+
+    it('should remove the label', () => {
+      expect(octokit.issues.removeLabel).toHaveBeenCalled();
+      expect(octokit.issues.removeLabel).toHaveBeenCalledWith({
+        issue_number: payload.issue.number,
+        name: 'good first issue',
+        owner: payload.repository.owner.login,
+        repo: payload.repository.name,
+      });
+    });
+  });
+
+  describe('check for good first issue by whitelisted user', () => {
+    beforeEach(async () => {
+      payload.sender.login = 'seanlip';
+      await dispatcher.dispatch('issues', 'labeled');
+    });
+
+    it('should be called for the payload', () => {
+      expect(checkIssueLabelModule.checkLabels).toHaveBeenCalled();
+    });
+
+    it('should not comment on issue', () => {
+      expect(octokit.issues.createComment).not.toHaveBeenCalled();
+    });
+
+    it('should not remove the label', () => {
+      expect(octokit.issues.removeLabel).not.toHaveBeenCalled();
+    });
+  });
+
+  describe('check for changelog labels', () => {
+    beforeEach(async () => {
+      payload.label.name = 'PR CHANGELOG: Server Errors -- @kevintab95';
+      await dispatcher.dispatch('issues', 'labeled');
+    });
+
+    it('should be called for the payload', () => {
+      expect(checkIssueLabelModule.checkLabels).toHaveBeenCalled();
+    });
+
+    it('should create appropriate comment', () => {
+      expect(octokit.issues.createComment).toHaveBeenCalled();
+
+      const user = payload.sender.login;
+      const link = 'here'.link(
+        'https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#' +
+        'labeling-issues-and-pull-requests');
+      const body = (
+        'Hi @' + user + ', changelog labels should not be used on issues.' +
+        ' I’m removing the label. You can learn more about labels ' + link);
+
+      expect(octokit.issues.createComment).toHaveBeenCalledWith({
+        issue_number: payload.issue.number,
+        body: body,
+        owner: payload.repository.owner.login,
+        repo: payload.repository.name,
+      });
+    });
+
+    it('should remove the label', () => {
+      expect(octokit.issues.removeLabel).toHaveBeenCalled();
+      expect(octokit.issues.removeLabel).toHaveBeenCalledWith({
+        issue_number: payload.issue.number,
+        name: 'PR CHANGELOG: Server Errors -- @kevintab95',
+        owner: payload.repository.owner.login,
+        repo: payload.repository.name,
+      });
+    });
+  });
+
+  describe('check for other PR labels', () => {
```

Comment:
Add a test for a valid label as well and ensure that the label isn't removed.

Reply:
Done.